app/controllers/api/subcollections/tags.rb

```diff
@@ -10,6 +10,15 @@ def assign_tags_resource(type, id, data)
         action_result(false, err.to_s)
       end
 
+      def unassign_tags_resource(type, id, data)
+        resource = resource_search(id, type, collection_class(type))
+        data['tags'].collect do |tag|
+          tags_unassign_resource(resource, type, tag['id'], tag)
+        end
+      rescue => err
```

Comment:
Is it possible that this could blow up while unassigning one of the tags, having already succeeded at some, and report a failure for all?